QModManager/QModManager.csproj

```diff
@@ -112,6 +108,12 @@
       <Private>False</Private>
     </Reference>
   </ItemGroup>
+  <ItemGroup Condition="'$(Configuration)|$(Platform)' == 'BZ.EXP|AnyCPU'">
```

Comment:
platform is not required here.
Condition="'$(Configuration)' == 'BZ.EXP'" should be sufficient.

Reply:
1b8543c